tests/downloaders/resources/tandfonline_1_out.json

```diff
@@ -1,25 +1,14 @@
 {
-  "journal": "Molecular Physics",
-  "ref": "doi:10.1080/00268976.2013.788745",
-  "volume": "111",
-  "subject": "Møller–Plesset perturbation theory; Epstein–Nesbet perturbation theory; divergences in perturbation theory; linked cluster theorem; coupled cluster",
-  "title": "The Møller–Plesset perturbation revisited: origin of high-order divergences",
-  "number": "9-11",
-  "publisher": "Taylor & Francis",
-  "pages": "1092-1099",
+  "abstract": "Contrarily to what happens with the Epstein–Nesbet (EN) zeroth-order Hamiltonian, the Møller–Plesset (MP) perturbation operator has diagonal matrix elements, the expression of which is recalled. It is a balance between hole–hole and particle–particle repulsions on one hand and of hole–particle attractions on the other hand. For the double excitations, which dominate the correlation effects, the attractive terms prevail and the second-order MP energy is underestimated, at least for atoms of the first rows of the periodic table. It will be shown that when the perturbation expansion reaches multiple excitations, the diagonal terms of the MP perturbation operator may become larger than the zeroth-order MP excitation energy and creates an oscillating divergence of the series. Several situations of this type will be presented. This divergence is linked to the non-additivity of excitation energies, while this additivity is an underlying assumption for the linked cluster theorem and the coupled cluster method. Th...",
   "author": " Jean-Paul   Malrieu  and  Celestino   Angeli ",
```

Comment:
This is still a bit annoying. In `fetch_data`, it gets a nicely formatted author string, but then that gets overwritten by the bibtex data [here](https://github.com/papis/papis/blob/v0.9/papis/downloaders/base.py#L168)

Reply:
I agree, we have to figure out a way of doing it in a controlled way, but I'm not sure yet what's best,
I think the bibtex information if the rest is good enough could be ignored.